src/shared/util/__tests__/logic.spec.ts

```diff
@@ -3,8 +3,6 @@ import { ObjectId } from 'bson-ext'
 import {
```

Comment:
I think we should move the relevant tests that have already been refactored to their own folder in `shared/utils/__tests__`!

Reply:
Sure, will have to move the remaining src/util/logic.ts functions too because it seems like the tests are dependant on them :D

Reply:
dependent tests can import from the new location; (or we can duplicate the logic if you meant the setup and teardown is reused), but you can also move them if you'd like

Reply:
I realized that all of the tests there are still dependent on some backend Mongo type `IFormDocument`, `IShowFieldsLogicSchema`, etc, so they still can't be brought to the root shared folder. Do you want me to refactor the tests too?

Reply:
I realized that all of the tests there are still dependent on some backend Mongo type `IFormDocument`, `IShowFieldsLogicSchema`, etc, so they still can't be brought to the root shared folder. Do you want me to refactor the tests too?